docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1767 @@
+{
```

Comment:
<p>That edge case is a great proof but I worry it may concern users/ start less useful discussions in relation to this notebook goals</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>mmm Ok. I do like it as a "proof" of the graph simplification. But I can of course remove it :)</p>

Reply:
<p>specially --&gt; specify</p>

Reply:
<p>But yeah, I don't think it adds much, agreeing with Ricardo.</p>